packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -1,6 +1,82 @@
 ---
 title: Okta API Products release notes 2021
 ---
+## December
+
+### Monthly release 2021.12.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Okta Sign-In Widget, version 5.14.0](#okta-sign-in-widget-version-5-14-0) | December 8, 2021 |
+| [Browser-like devices can now use the Device Authorization Grant flow](#browser-like-devices-can-now-use-the-device-authorization-grant-flow) | December 8, 2021 |
+| [Custom domains with Okta-managed certificates](#custom-domains-with-okta-managed-certificates) | December 8, 2021 |
+| [Device Authorization grant type is now GA in Production](#device-authorization-grant-type-is-now-ga-in-production) | November 3, 2021 |
+| [Dynamic Issuer Mode is GA in Preview](#dynamic-issuer-mode-is-ga-in-preview) | December 8, 2021 |
+| [Error response updated for malicious IP address sign-in requests](#error-response-updated-for-malicious-ip-address-sign-in-requests) | December 8, 2021 |
+| [IdP discovery supported for Device Authorization Grant flow](#idp-discovery-supported-for-device-authorization-grant-flow) | December 8, 2021 |
+| [List Schemas operation for the Log Streaming API now available](#list-schemas-operation-for-the-log-streaming-api-now-available) | December 8, 2021 |
+| [The word "source" is now allowed with custom application username formats](#the-word-source-is-now-allowed-with-custom-application-username-formats) | December 8, 2021 |
+| [Upload Logo for Org deprecated](#upload-logo-for-org-deprecated) | December 8, 2021          |
+| [User Types API and Mappings API support OAuth 2.0](#user-types-api-and-mappings-api-support-oauth-2-0) | December 8, 2021 |
+| [Bugs fixed in 2021.12.0](#bugs-fixed-in-2021-12-0)               | December 8, 2021          |
+
+#### Okta Sign-In Widget, version 5.14.0
+
+For details about this release, see the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.14.0). For more information about the Widget, see the Okta [Sign-In Widget guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-449551-->
```

Comment:
```suggestion
See the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.14.0) for more information about this release. See the Okta [Sign-In Widget guide](/code/javascript/okta_sign-in_widget/) for more information about the Widget. <!--OKTA-449551-->
```

Reply:
Are we changing the blurb that we reuse each time? This is what is in our wiki? I'm not opposed - but we need to update the [How to work on weekly and monthly API Release Notes](https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/1707806413/How+to+work+on+weekly+and+monthly+API+Release+Notes) doc.
<img width="1270" alt="Screen Shot 2021-12-07 at 4 35 55 PM" src="https://user-images.githubusercontent.com/45509385/145122555-cbfd443c-daef-4cfb-b40f-8dd27c995d25.png">

Reply:
Oh right. Leave as is then.